src/main/kotlin/racingcar/DriveRule.kt

```diff
@@ -1,17 +1,18 @@
 package racingcar
 
-import java.util.concurrent.ThreadLocalRandom
+import kotlin.random.Random
 
 class DriveRule : DriveRangeStrategy {
     companion object {
-        private const val FORWARD_NUMBER = 4
+        private const val NUMBER_RANDOM_MAX = 9
+        const val FORWARD_NUMBER = 4
+    }
 
-        fun random(): Int {
-            return ThreadLocalRandom.current().nextInt(0, 9)
-        }
+    fun random(): Int {
+        return Random.nextInt(NUMBER_RANDOM_MAX)
```

Comment:
추가로 공유드려요. 랜덤값의 범위를 줄수있는 방식입니다.
```suggestion
        return (0.. NUMBER_RANDOM_MAX).random()
```